beets/importer.py

```diff
@@ -547,9 +547,22 @@ def _emit_imported(self, lib):
         plugins.send('album_imported', lib=lib, album=self.album)
 
     def emit_created(self, session):
-        """Send the `import_task_created` event for this task.
+        """Send the `import_task_created` event for this task and return a list
+        of tasks which might be returned by plugins.
```

Comment:
A little more detail in this docstring would be helpful: for example: "Return a list of tasks that should continue through the pipeline. By default, this is a list containing only the task itself, but plugins can replace the task with new ones."